_topics/en/swap-in-potentiam.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Swap-in Potentiam (SIP)
+
+## Optional. test Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: swap-in potentiam
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Swap-in-Potentiam (SIP)** is a protocol designed to enable the immediate transfer
+  of already-confirmed onchain Bitcoin funds to the Lightning Network, lowering
+  trust requirements by precommitting to an LSP and incorporating a timeout
+  mechanism for unilateral access.
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+  - title: Moving Onchain Funds "Instantly" To Lightning (Jesse Posner, ZmnSCPxj)
+    link: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-January/003810.html
+
+  - title: LSPS C= Extension, Swap-in-Potentiam
+    link: https://github.com/ZmnSCPxj-jr/swap-in-potentiam/blob/master/doc/swap-in-potentiam.md
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "Swap-in Potentiam Announced"
+    url: /en/newsletters/2023/12/20/#swap-in-potentiam
+
+  - title: Non-interactive LN channel open commitments
+    url: /en/newsletters/2023/01/11/#non-interactive-ln-channel-open-commitments
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Hash Time Locked Contracts from Bitcoin Wiki
+    link: https://en.bitcoin.it/wiki/Hash_Time_Locked_Contracts
```

Comment:
Is there a special reason to link to HTLCs on the wiki here rather than our [HTLCs topic](https://bitcoinops.org/en/topics/htlc/) (which itself links to the wiki)?